src/Mappers/LowerCaseMapper.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We do not add strict types to Spatie packages

Reply:
Got it. My file templates use strict mode by default.

I'll remove that.